x/pairing/keeper/filters/geolocation_filter.go

```diff
@@ -6,18 +6,19 @@ import (
 	planstypes "github.com/lavanet/lava/x/plans/types"
 )
 
-// TODO: This is a temp filter until the geolocation mechanism changes (this is not optimal)
+// TODO: This is a filter is disabled (InitFilter always returns false)
+// will be used in the future to exclude geolocations (thus keeping this code)
 
 type GeolocationFilter struct {
 	geolocation uint64
 }
 
 func (f *GeolocationFilter) InitFilter(strictestPolicy planstypes.Policy) bool {
-	if strictestPolicy.SelectedProvidersMode == planstypes.SELECTED_PROVIDERS_MODE_DISABLED ||
-		strictestPolicy.SelectedProvidersMode == planstypes.SELECTED_PROVIDERS_MODE_ALLOWED {
-		f.geolocation = strictestPolicy.GeolocationProfile
-		return true
-	}
+	// if strictestPolicy.SelectedProvidersMode == planstypes.SELECTED_PROVIDERS_MODE_DISABLED ||
```

Comment:
I prefer doing it my way because it requires minimal changes to the code (commenting out this code will make me change filter.go)

Reply:
I mean, you comment out 4 lines inside this function. I assume it's temporary - otherwise you'd remove them entirely. Using "/* ... */" is useful in Go to comment out a block:
```
/*
        if strictestPolicy.SelectedProvidersMode == planstypes.SELECTED_PROVIDERS_MODE_DISABLED ||
		strictestPolicy.SelectedProvidersMode == planstypes.SELECTED_PROVIDERS_MODE_ALLOWED {
		f.geolocation = strictestPolicy.GeolocationProfile
		return true
	}
*/
```

Reply:
done. see https://github.com/lavanet/lava/commit/22b00fd0cc285f226d02c432dab6fed1abe0b171